### DISCUSSION TITLE
Font Resources

### DISCUSSION THREAD

**Discussion by papwuj** [UI5/webcomponents-react#5573] [Q&A]
Hello teachers, how can I set the resources in the rectangle box to access the font resources in the local project without accessing the Internet resources, so that the deployment to the intranet resources are displayed normally.

![image](https://github.com/SAP/ui5-webcomponents-react/assets/16931184/a8b08f97-2795-44e0-8a2a-cf0a2a5121af)

**Comment by Lukas742** [comment]
Hi @ruoyaya 

the font styles are fetched by the ui5-webcomponents, you can download the files from their sources inside the network tab.

You can find out more about ui5-webcomponents fonts [here](https://sap.github.io/ui5-webcomponents/playground/?path=/docs/docs-customizing-fonts--docs).
